tutorials/eda_contents/Computing_Environment/Unix.md

```diff
@@ -0,0 +1,363 @@
+- An absolute path specifies a location from the root of the file system.
+- Directory names in a path are separated with '/' on Unix, but '\\\\' on Windows.
+- '..' means 'the directory above the current one'; '.' on its own means 'the current directory'.
+- Most files' names are `something.extension`. The extension isn't required, and doesn't guarantee anything, 
+but is normally used to indicate the  type of data in the file.
+- Most commands take options (flags) which begin with a '-'.
+
+## Working with Files and Directories
+
+You would have seen basic ways to navigate through files and directories in Unix. Now, let's make them.
+
+To create a directory, we use the command `mkdir` plus the name of the directory you wish to create.
+
+~~~
+/notebook_dir/mypublic$ mkdir example_of_directory
+/notebook_dir/mypublic$ ls
+'Environment'   example_of_directory   Marble.md   Plan.ipynb
+~~~
+
+Note the `mkdir` yielded no output but when we check for the list of files and directories within the current working directory, 
+we see `example_of_directory` was created. 
+Be aware that the `mkdir` command makes the new directory within your current working directory. Ensure you create the directory in 
+the correct location.
+
+## Good names for files and directories
+
+ Complicated names of files and directories can make your life painful
+ when working on the command line. Here we provide a few useful
+ tips for the names of your files.
+
+ 1. Don't use whitespaces.
+
+   Whitespaces can make a name more meaningful but since whitespace is used to break arguments on the command line
+   is better to avoid them on name of files and directories. You can use `-` or `_` instead of whitespace.
+
+ 2. Don't begin the name with `-` (dash).
+
+    Commands treat names starting with `-` as options.
+
+ 3. Stick with letters, numbers, `.` (period), `-` (dash) and `_` (underscore).
+
+    Many other characters have special meanings on the command line. We will learn
+    about some of these during this lesson. There are special characters that can
+    cause your command to not work as expected and can even result in data loss.
+
+ If you need to refer to names of files or directories that have whitespace
+ or another non-alphanumeric character, you should surround the name in quotes (`""`).
+
+ Now, let's go to the new directory we have created.
+
+Assuming you made a directory called `example_of_directory`, then type:
+
+~~~
+$ cd ex<TAB>
+~~~
+
+This will auto-input `example_of_directory`.
+
+~~~
+$ pwd
+/notebook_dir/mypublic/example_of_directory
+~~~
+Let's create a file called `new_file.txt` in this new dictionary using the command `touch`.
+
+~~~
+/example_of_directory$ touch new_file.txt
+/example_of_directory$ ls
+new_file.txt
+~~~
+
+Now, let's see how we can remove any files using the `rm` command:
+
+~~~
+$ rm new_file.txt
+$ ls
+$ 
+~~~
+
+The `ls` command now returns no output. 
+
+#### Deleting Is Forever
```

Comment:
Maybe put this section in a warning block https://jupyterbook.org/en/stable/content/content-blocks.html

Reply:
added warning block